examples/responses/background_tasks_1.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from starlite import BackgroundTask, Response, Starlite, get
+
+logger = logging.getLogger(__name__)
+
+
+async def logging_task(identifier: str, message: str) -> None:
+    logger.info("%s: %s", identifier, message)
+
+
+@get("/")
+def greeter(name: str) -> Response[dict[str, str]]:
```

Comment:
Use Python 3.8 compatible type annotations:

```suggestion
def greeter(name: str) -> Response[Dict[str, str]]:
```